### ISSUE TITLE
DNS resolution is broken after upgrade from 1.1.4 to 1.2.0-pre4-rc10

### ISSUE THREAD

**Issue by soumyalj** [rancher/rancher#6699]
**Rancher Version:**: 1.1.4 to 1.2.0-pre4-rc10

**Docker Version:**1.10.3

**OS and where are the hosts located? (cloud, bare metal, etc):**Ubuntu, Cloud

**Setup Details: (single node rancher vs. HA rancher, internal DB vs. external DB)**Single node

**Environment Type: (Cattle/Kubernetes/Swarm/Mesos)**: Cattle

**Steps to Reproduce:**
1. Create a setup in v1.1.4
2. Upgrade to v1.2.0-pre4-rc10
3. Upgrade the env

**Results:** 
DNS resolution is broken
```
root@c54f457110d8:/# ping www.google.com
ping: unknown host
root@c54f457110d8:/# ping myalias
ping: unknown host
root@c54f457110d8:/# dig myalias

; <<>> DiG 9.8.4-rpz2+rl005.12-P1 <<>> myalias
;; global options: +cmd
;; connection timed out; no servers could be reached
root@c54f457110d8:/# dig newalias

; <<>> DiG 9.8.4-rpz2+rl005.12-P1 <<>> newalias
;; global options: +cmd
;; connection timed out; no servers could be reached
```

**Comment by deniseschannon** [comment]
Fixed in network-manager - v0.2.6

**Comment by soumyalj** [comment]
Tested with upgrade from 1.1.4 to 1.2.0-pre4-rc11. DNS resolution is functional.
```
root@76210b2ef827:/# ping www.google.com
PING www.google.com (74.125.193.147): 48 data bytes
56 bytes from 74.125.193.147: icmp_seq=0 ttl=56 time=1.737 ms
56 bytes from 74.125.193.147: icmp_seq=1 ttl=56 time=0.912 ms
56 bytes from 74.125.193.147: icmp_seq=2 ttl=56 time=0.857 ms
^C--- www.google.com ping statistics ---
3 packets transmitted, 3 packets received, 0% packet loss
round-trip min/avg/max/stddev = 0.857/1.169/1.737/0.402 ms
root@76210b2ef827:/# dig myalias +short
10.42.42.157
10.42.139.146

```
